swagger.yaml

```diff
@@ -751,7 +751,99 @@ paths:
                 items:
                   $ref: "#/components/schemas/Error"
 
-  "/vacationsRequests":
+  "/severa/projects/{severaProjectGuid}/phaseswithhierarchy":
+    get:
+      operationId: getPhasesBySeveraProjectId
+      summary: Get all the phases for project
+      description: Retrieve all the phases for a particular project with given ID.
+      tags:
+        - Phases
+      parameters:
+        - name: severaProjectGuid
+          in: path
+          required: true
+          description: ID of the project.
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Phases entry retrieved successfully
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Phase"
+        "default":
+          description: Invalid request was sent to the server.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Error"
+
+  "/severa/{severaUserGuid}/resourceallocations":
+    get:
+      operationId: getAllocationsBySeveraUserGuid
+      summary: Get the resource allocations for a user.
+      description: Retrieve the resource allocations for a user based on their Severa GUID.
+      tags:
+        - ResourceAllocations
+      parameters:
+        - name: severaUserGuid
+          in: path
+          required: true
+          description: severaGuid of the user.
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Resource allocations retrieved successfully
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/ResourceAllocations"
+        "default":
+          description: Invalid request was sent to the server.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Error"
+
+  "/severa/users/{severaUserGuid}/workhours":
+    get:
+      operationId: getWorkHoursBySeveraProjectId
+      summary: Get all the work hours for project
+      description: Retrieve all the work hours for a particular user with given ID.
+      tags:
+        - WorkHours
+      parameters:
+        - name: severaUserGuid
+          in: path
+          required: true
+          description: ID of the work hour.
```

Comment:
Isn't it the ID of the user?

Reply:
Yes, it is. We will fix the description.